app/views/users/show.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= render partial: 'users/user', locals:{user:@user} -%>
+<div class="user-bio">
+  <label class="bio-label">bio</label>
+  <p><%= @user.bio -%></p>
+</div>
+<h3>Posts</h3>
+<% @user.posts.each do |post| %> <%= render partial: 'posts/post',
```

Comment:
- [ ] Here, you are supposed to display the most recent posts using the method you created in your user model. It should be `@users.most_recent.each`